test/helpers/dom/RecoveryQuestionForm.js

```diff
@@ -3,6 +3,7 @@ define(['./Form'], function (Form) {
   var ANSWER_FIELD = 'answer';
   var SHOW_ANSWER_FIELD = 'showAnswer';
   var CLASS_SELECTOR = '.recovery-question';
+  var BACK_TO_LOGIN_BTN = 'back-button';
```

Comment:
Hmm, actually never mind; I see we are already doing this [here](https://github.com/okta/okta-signin-widget/blob/master/test/spec/UnlockAccount_spec.js#L196)